lib/journey.ex

```diff
@@ -0,0 +1,145 @@
+defmodule Journey.Step do
```

Comment:
Since it's a lib, it'd be nice to have `@moduledoc` for modules and `@doc` for public functions.

Reply:
In the future ;)